examples/ovsbridge_create.yml

```diff
@@ -3,6 +3,14 @@ interfaces:
   - name: eth1
     type: ethernet
     state: up
+  - name: ovs0
```

Comment:
IT seems to me that it would be easier to read if the internal port definition is specified after the actual bridge. The eth1 entry should probably be moved for consistency as well.

Reply:
Ok, but lets do it in a flowing PR and follow this pattern in all examples.